php.ini

```diff
@@ -0,0 +1,4 @@
+upload_max_filesize = 2000M
+post_max_size = 2000M
+memory_limit = -1
```

Comment:
exports & imports can take a very long time, not sure what would be enough and reasonnable